src/net-socket/helpers/common.ts

```diff
@@ -0,0 +1,42 @@
+export function getRandomArbitrary(min: number, max: number) {
+  return Math.random() * (max - min) + min;
+}
+
+export async function delayRnd(minSec: number, maxSec: number) {
+  const delaySeconds = getRandomArbitrary(minSec * 1000, maxSec * 1000);
+  return new Promise((resolve) => {
+    setTimeout(resolve, delaySeconds);
+  });
```

Comment:
```suggestion
  return delay(delaySeconds);
```

у тебя уже есть хелпер для ожидания, переиспользуй его